libcxx/include/fstream

```diff
@@ -188,6 +195,7 @@ typedef basic_fstream<wchar_t> wfstream;
 #include <__utility/move.h>
 #include <__utility/swap.h>
 #include <__utility/unreachable.h>
+#include <cassert>
```

Comment:
We should not include asserts in headers. At least one of the CI jobs fails due this.